meddlr/data/transforms/transform.py

```diff
@@ -11,6 +11,7 @@
 from meddlr.utils import transforms as T
 
 from .motion import MotionModel
+from .motion import MotionModel2D
```

Comment:
use absolute paths, not relative paths, for imports - eg. `from meddlr.data.transforms.motion import MotionModel2D`